tests/unit/core/test_evaluate_horner_polynomial.py

```diff
@@ -0,0 +1,15 @@
+"""Test of the evaluate_horner_polynomial function."""
+
+import numpy as np
+
+from pyrealm.core.utilities import evaluate_horner_polynomial
+
+
+def test_evaluate_horner_polynomial():
+    """Test the evaluate_horner_polynomial function."""
+    for deg in range(10):
+        cf = np.random.rand(deg + 1)
+        x = np.random.rand()
```

Comment:
Mostly just to check it all works with negative values 😄 Could equally use a uniform distribution in [-1, 1].

```suggestion
        cf = np.random.normal(deg + 1)
        x = np.random.normal()
```